cmd/fyne/internal/commands/package-unix.go

```diff
@@ -67,9 +68,11 @@ func (p *Packager) packageUNIX() error {
 			return fmt.Errorf("failed to write Makefile string: %w", err)
 		}
 
+		buf := bytes.Buffer{}
 		tarCmd := execabs.Command("tar", "-Jcf", p.name+".tar.xz", "-C", tempDir, "usr", "Makefile")
+		tarCmd.Stderr = &buf
 		if err = tarCmd.Run(); err != nil {
-			return fmt.Errorf("failed to create archive with tar: %w", err)
+			return fmt.Errorf("failed to create archive with tar: %s - %w", buf.Bytes(), err)
```

Comment:
nit: Again, might not make a difference
```suggestion
			return fmt.Errorf("failed to create archive with tar: %s - %w", buf.String(), err)
```